src/components/Button.js

```diff
@@ -112,6 +113,26 @@ const StyledButton = styled.a`
     :focus {
       box-shadow: 0 0 0 .2rem ${hexToRgba(p.theme.colors.primary, 0.5)};
     }`}
+    ${p =>
+    p.color !== 'primary' &&
+    p.color !== 'secondary' &&
+    p.color !== 'tertiary' && // some color other than the variants
+    `color: ${p.labelColor ?? p.theme.colors.background};
+      background: ${p.color};
+      :hover {
+        ${
+          p.disabled
+            ? `
+        cursor: not-allowed;
+        `
+            : `
+        background: ${p.hover ?? p.theme.colors.tertiaryHover};
```

Comment:
I think we've mostly been using `||`, could we use that here as well?

Reply:
yupp